internal/pkg/cli/task_run.go

```diff
@@ -965,12 +967,18 @@ func (o *runTaskOpts) buildAndPushImage() error {
 	if o.dockerfileContextPath != "" {
 		ctx = o.dockerfileContextPath
 	}
-
-	if _, err := o.repository.BuildAndPush(&dockerengine.BuildArguments{
+	buildArgs := &dockerengine.BuildArguments{
+		URI:        uri,
```

Comment:
So this struct is missing some of its fields because they aren't supported in task run, right?